spec/game_spec.rb

```diff
@@ -0,0 +1,118 @@
+require './lib/game'
+
+describe Game do
```

Comment:
Great extensive testing of this entire class! Makes for a great robust programme